src/vsr/replica.zig

```diff
@@ -1065,7 +1065,14 @@ pub fn ReplicaType(
         };
 
         /// NOTE: self.superblock must be initialized and opened prior to this call.
-        fn init(self: *Replica, allocator: Allocator, options: Options) !void {
+        fn init(
+            self: *Replica,
+            allocator: Allocator,
+            storage: *Storage,
+            message_pool: *MessagePool,
+            time: Time,
```

Comment:
Likewise, I would order from time -> storage -> message_pool. (Most to least general.)